### ISSUE TITLE
Add whole album to favorite or playlist

### ISSUE THREAD

**Issue by narFnarF** [resonatecoop/stream2own#279]
### Operating system
Win 10

### Browser name and version
Firefox 106

### Expected behavior
A way to add the whole album to my favorites or to a playlist

### Actual behavior
No way to add the full album, other than adding each song manually.

### Steps to reproduce behavior
1. Go to an album page
2. Click the "..." menu

**Comment by simonv3** [comment]
Hey @narFnarF. This is a great idea and has been discussed a [couple](https://community.resonate.coop/t/favourite-an-album/2704) [of](https://community.resonate.coop/t/user-music-library-favoriting-following/1264/2?u=psi) [times](https://community.resonate.coop/t/being-able-to-add-artists-to-favorites/2327) before on the community forum. In the legacy API there is actually a PR that's outstanding for to enable it on the back-end! I can't link it because it's private, but it's real 😓 . 

But that's been put on hold a bit as we consolidate all of Resonate's infrastructure. This is a long process but we're [making good progress](https://community.resonate.coop/t/the-consolidated-api-progress-and-updates/3287). Once all of this is done, this is definitely one of the top tickets.

**Comment by hsandt** [comment]
This would be very useful! I added a related issue to show if a track is already in favorites or not, to complement this feature: https://github.com/resonatecoop/stream/issues/282
